contrib/test/integration/test.yml

```diff
@@ -12,14 +12,14 @@
     extra_storage_opts: " --storage-opt overlay.override_kernel_check=1"
   when: ansible_distribution == 'RedHat' or ansible_distribution == 'CentOS'
 
-- name: ensure directory exists for e2e reports
+- name: ensure directory exists for integration results
   file:
     path: "{{ artifacts }}"
     state: directory
 
 - name: run integration tests
-  shell: "CGROUP_MANAGER=cgroupfs STORAGE_OPTS='--storage-driver=overlay{{ extra_storage_opts | default('') }}' make localintegration >& {{ artifacts }}/testout.txt"
```

Comment:
Done...and I moved the ``extra_storage_opts`` definition into ``vars.yml``.  Because looking back and forth trying to figure out your suggestion was making my neck hurt 😀